sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -34,9 +35,11 @@ export interface AnalyzeSentimentSuccessResult extends TextAnalyticsSuccessResul
 }
 
 // @public
-export class CognitiveServicesCredential extends ApiKeyCredentials {
+export class CognitiveServicesCredential implements ServiceClientCredentials {
```

Comment:
Per the updated guidelines, we'll call this TextAnalyticsSubscriptionKeyCredential.  In .NET we don't want to inherit from an azure core or azure identity type, but this may be a language-dependent decision.

Reply:
This isn't inheritance anymore; it asserts that this class implements the expected interface in order to participate in the request pipeline.